### ISSUE TITLE
Kubernetes deployment

### ISSUE THREAD

**Issue by ribbybibby** [ribbybibby/ssl_exporter#14]
I should provide a kustomize base/helm chart (or both?) as an upstream for others to use and contribute to.

**Comment by andrii29** [comment]
Hello @ribbybibby,

any updates?

Regards,
Andrii

**Comment by ribbybibby** [comment]
Hi @andrii29. I haven't got around to doing this yet. Here's an example service and deployment you could refer to though: https://github.com/ribbybibby/ssl_exporter/issues/12#issuecomment-562966945.

**Comment by jgrossmac** [comment]
Yeah I can help out with this as well.  I've done one for our setup and it works well.

**Comment by MatusJenca2** [comment]
Hello. 
We at [dNation](https://github.com/dNationCloud)  recently needed a ssl exporter for our [kubernetes monitoring stack](https://github.com/dNationCloud/kubernetes-monitoring-stack), so I have created a [helm char](https://github.com/dNationCloud/ssl-exporter)t for your ssl-exporter. Make sure to check that out if you are interested. Check out our [monitoring stack](https://github.com/dNationCloud/kubernetes-monitoring-stack) as well.

**Comment by Bak3y** [comment]
FYI above chart is a daemonset, and definitely not how I'd want to deploy this.

**Comment by Bak3y** [comment]
Also, helm chart please.
